### PR TITLE
Document TLS requirement for host cert with SSL (SOFTWARE-5240) 

### PR DESCRIPTION
**Pull request by edquist**

### CODE REVIEW

File: docs/compute-element/install-htcondor-ce.md [hunk 1]
```diff
@@ -29,8 +29,11 @@ Before starting the installation process, consider the following points, consult
     `gratia`
     You will also need to create Unix accounts for each collaboration that you wish to support.
     See details in the ['Configuring authentication' section below](#configuring-authentication).
--   **SSL certificate:** The HTCondor-CE service uses a host certificate at `/etc/grid-security/hostcert.pem` and an
-    accompanying key at `/etc/grid-security/hostkey.pem`
+-   **SSL certificate:** The HTCondor-CE service uses a host certificate named
+    `hostcert.pem` and an accompanying key named `hostkey.pem`.
+    -  If using a Let's Encrypt cert, place the cert and key into `/etc/pki/tls`
+    -  If using an IGTF cert, place these under `/etc/grid-security`
```
Comment (brianhlin) [inline docs/compute-element/install-htcondor-ce.md:35]: We should mention the exact paths, `/etc/grid-security/host{cert,key}.pem` (but spelled out)

File: docs/compute-element/install-htcondor-ce.md [hunk 2]
```diff
@@ -29,8 +29,11 @@ Before starting the installation process, consider the following points, consult
     `gratia`
     You will also need to create Unix accounts for each collaboration that you wish to support.
     See details in the ['Configuring authentication' section below](#configuring-authentication).
--   **SSL certificate:** The HTCondor-CE service uses a host certificate at `/etc/grid-security/hostcert.pem` and an
-    accompanying key at `/etc/grid-security/hostkey.pem`
+-   **SSL certificate:** The HTCondor-CE service uses a host certificate named
+    `hostcert.pem` and an accompanying key named `hostkey.pem`.
+    -  If using a Let's Encrypt cert, place the cert and key into `/etc/pki/tls`
```
Comment (brianhlin) [inline docs/compute-element/install-htcondor-ce.md:34]: Specifically, the cert/key should match the default HTCondor-CE (and HTCondor) values:

```
AUTH_SSL_SERVER_CERTFILE = /etc/pki/tls/certs/localhost.crt
AUTH_SSL_SERVER_KEYFILE = /etc/pki/tls/private/localhost.key
```

File: docs/security/host-certs/overview.md [hunk 3]
```diff
@@ -33,23 +55,10 @@ We recommend requesting host certificates from one of the following CAs:
...
-issuer=/DC=org/DC=cilogon/C=US/O=CILogon/CN=CILogon OSG CA 1
-notBefore=Jan  4 21:08:09 2010 GMT
-notAfter=Jan  4 21:08:09 2011 GMT
-```
-
-If you are using OpenSSL 1.1, you may notice minor formatting differences.
 
+!!!note
+    For SSL to work properly, you will need to request a host certificate with
+    "TLS Web Server Authentication" included in the X509v3 Extended Key Usage.
```
Comment (brianhlin) [inline docs/security/host-certs/overview.md:61]:
```suggestion
    "TLS Web Server Authentication" and "TLS Web Client Authentication" included
    in the X509v3 Extended Key Usage.
```

Come to think of it (I know it's not noted in the original ticket), users probably also need "TLS Web Client Authentication"

**Review by brianhlin — APPROVED** [review]
Mostly LGTM, the specific host cert/key paths need to be updated, though

File: docs/compute-element/install-htcondor-ce.md (hunk 1 above)
Comment (edquist) [reply docs/compute-element/install-htcondor-ce.md]: Ok, updated.

File: docs/compute-element/install-htcondor-ce.md (hunk 2 above)
Comment (edquist) [reply docs/compute-element/install-htcondor-ce.md]: Ah, tricky; updated again.

**Review by brianhlin — APPROVED** [review]
Huzzah!

**Pull request merged**